vignettes/cookbook_overview.Rmd

```diff
@@ -14,6 +14,41 @@ knitr::opts_chunk$set(
 )
 ```
 
-```{r setup}
-library(pacta.loanbook)
-```
+# Overview
+
+This cookbook provides a step-by-step guide to running the PACTA for Banks analysis using the `r2dii.*` suite of packages. The analysis is designed to help banks assess the alignment of their loan books with the Paris Agreement goals.
+
+The sections of the cookbook are as follows:
+
+1. **Overview**: This section provides an overview of the PACTA for Banks analysis and its main steps.
+2. **[Preparatory Steps](cookbook_preparatory_steps.html)**: Prepare the ABCD data and optionally a custom sector split.
```

Comment:
```suggestion
2. **[Preparatory Steps](cookbook_preparatory_steps.html)**: Prepare the Loanbook, ABCD and Scenario input data.
```

Reply:
isn't it a "loan book" rather than a "Loanbook"?